web/src/routes/OffersPage.tsx

```diff
@@ -40,30 +38,24 @@ const OffersPage = ({ cityModel, cityCode, languageCode, languages }: CityRouteP
   const { data: offers, loading, error: offersError } = useLoadFromEndpoint(requestOffers)
 
   const toTileModels = useCallback(
-    (offers: Array<OfferModel>): Array<TileModel> => {
-      const tiles = offers.map(offer => {
+    (offers: Array<OfferModel>): Array<TileModel> =>
+      offers.map(offer => {
         let path = offer.path
 
         if (offer.alias === SPRUNGBRETT_OFFER) {
+          // the url stored in the sprungbrett offer is the url of the endpoint
           path = pathnameFromRouteInformation({ route: SPRUNGBRETT_OFFER_ROUTE, cityCode, languageCode })
+        } else if (offer.alias === SHELTER_ROUTE) {
+          path = pathnameFromRouteInformation({ route: SHELTER_ROUTE, cityCode, languageCode })
         }
 
         return new TileModel({
-          title: offer.title,
-          // the url stored in the sprungbrett offer is the url of the endpoint
+          title: t(offer.title),
           path,
           thumbnail: offer.thumbnail,
           postData: offer.postData
         })
-      })
-
-      if (shelterOfferEnabled(cityCode)) {
-        const path = pathnameFromRouteInformation({ route: SHELTER_ROUTE, cityCode, languageCode })
-        tiles.push(new TileModel({ title: t('shelter:title'), path, thumbnail: shelterIcon }))
-      }
-
-      return tiles
-    },
+      }),
```

Comment:
Do we show somewhere else this tile?

Reply:
See comment below, it is now delivered by the cms as offer.